types/ember-cli-fastboot/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "paths": {
+            "@ember/*": ["ember__*"]
+        },
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts",
```

Comment:
Per suggestion above:

```suggestion
        "index.d.ts",
        "-private.d.ts",
```

Reply:
per my earlier changes:
```Error: ember-cli-fastboot: Only index.d.ts may be listed explicitly in tsconfig's "files" entry.```

DT prefers you just import the file in your index.d.ts, which I've done

Reply:
Well that’s an interesting rule. 👍🏼